config/locales/en.json

```
@@ -502,7 +491,7 @@
   "dashboard.consent": "has given consent for Service Canada to obtain personal information about them.",
   "dashboard.view_download_consent": "View their consent",
```

```suggestion
  "dashboard.view_download_consent": "View and print their consent  for your records",
```
Changed this too, because I'm difficult like that.